monitor/pom.xml

```diff
@@ -0,0 +1,159 @@
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}/../iotdb/iotdb/lib_cluster</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <outputDirectory>${project.basedir}/../iotdb/iotdb/lib_cluster</outputDirectory>
+                </configuration>
+            </plugin>
+            <!--using `mvn test` to run UT, `mvn verify` to run ITs
+                  Reference: https://antoniogoncalves.org/2012/12/13/lets-turn-integration-tests-with-maven-to-a-first-class-citizen/-->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skipTests>${monitor.ut.skip}</skipTests>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>run-integration-tests</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <skipTests>${monitor.test.skip}</skipTests>
+                    <skipITs>${monitor.it.skip}</skipITs>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <id>skipMonitorTests</id>
+            <activation>
+                <property>
+                    <name>skipTests</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <monitor.test.skip>true</monitor.test.skip>
+                <monitor.ut.skip>true</monitor.ut.skip>
+                <monitor.it.skip>true</monitor.it.skip>
+            </properties>
+        </profile>
+        <profile>
+            <id>skipUT_Monitor_Tests</id>
+            <activation>
+                <property>
+                    <name>skipUTs</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <iotdb.ut.skip>true</iotdb.ut.skip>
```

Comment:
iotdb.ut.skip?

Reply:
monitor.ut.skip?

Reply:
Because in cluster/pom it is iotdb.ut.skip, so should change iotdb.ut.skip to cluster.ut.skip in cluster/pom as well?